File: social-service/src/main/java/com/eirsteir/coffeewithme/social/dto/UserProfile.java
package com.eirsteir.coffeewithme.social.dto;

import com.eirsteir.coffeewithme.social.domain.university.University;
import lombok.*;
import lombok.experimental.Accessors;

@Getter
@Setter
@ToString
@Builder
@Accessors(chain = true)
@NoArgsConstructor
@AllArgsConstructor
public class UserProfile {

    private Long id;
    private String email;
    private String nickname;
    private String name;
    private Integer friendsCount;
    private University university;

}

File: social-service/src/main/kotlin/com/eirsteir/coffeewithme/social/dto/UserProfile.kt
package com.eirsteir.coffeewithme.social.dto

import com.eirsteir.coffeewithme.social.domain.university.University
import lombok.Builder
import lombok.Getter
import lombok.Setter
import lombok.experimental.Accessors

data class UserProfile(
    val id: Long,
    val email: String,
    val nickname: String,
    val name: String,
    var friendsCount: Int,
    val university: University,
)